homeassistant/core.py

```diff
@@ -1418,6 +1422,9 @@ async def async_call(
         else:
             processed_data = service_data
 
+        if target:
+            processed_data.update(target)
```

Comment:
Should it be like this or should it split out in service call too ?

Reply:
Before we check the schema it needs to be in the data, or all schemas need updating...